restore.go

```diff
@@ -50,14 +50,9 @@ var restoreCommand = cli.Command{
 			Usage: "cgroups mode: 'soft' (default), 'full' and 'strict'.",
 		},
 		cli.StringFlag{
-			Name:  "config-file, c",
-			Value: "config.json",
-			Usage: "path to spec file for writing",
-		},
-		cli.StringFlag{
-			Name:  "runtime-file, r",
-			Value: "runtime.json",
-			Usage: "path for runtime file for writing",
+			Name:  "bundle-dir, b",
```

Comment:
Having just `bundle` will help with overloading this for bundle files as well in the future.